### PR TITLE
Custom params 2

### PR DESCRIPTION
**Pull request by iman-nebulr**

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (iman-nebulr) [inline .devcontainer/devcontainer.json:1]: chech this file

File: package.json [hunk 1]
```diff
@@ -31,28 +31,25 @@
...
-    "@nestjs/config": "^0.5.0 || ^1.0.2",
-    "@nestjs/core": "^7.0.0 || ^8.1.2",
-    "@nestjs/graphql": "^7.6.0 || ^8.0.2",
-    "@nestjs/mongoose": "^7.2.0 || ^8.0.1",
+    "@nebulr-group/nblocks-ts-client": "^5.4.2",
+    "@nestjs/common": "^8.4.7",
+    "@nestjs/config": "^1.0.2",
+    "@nestjs/core": "^8.4.7",
+    "@nestjs/graphql": "^8.0.2",
+    "@nestjs/mongoose": "^9.2.0",
```
Comment (orr3) [inline package.json:46]: This will most certainly have implications on apps that use this plugin. e.g. Northwhistle.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nebulr-group/nblocks-nestjs",
-  "version": "5.1.0",
+  "version": "5.2.1",
```
Comment (orr3) [inline package.json:3]: Breaking changes, version `6.0.0` would be more appropriate

File: package.json [hunk 3]
```diff
@@ -108,5 +107,10 @@
     "moduleNameMapper": {
       "^src/(.*)$": "<rootDir>/$1"
     }
+  },
+  "overrides": {
```
Comment (orr3) [inline package.json:111]: Let's go through what this means some other time

File: src/tenant/tenant.graphql-model.ts [hunk 4]
```diff
@@ -1,174 +1,181 @@
...
   email: string;
 
-  @Field(type => String, { nullable: true })
+  @Field((type) => String, { nullable: true })
   firstName?: string;
 
-  @Field(type => String, { nullable: true })
+  @Field((type) => String, { nullable: true })
   lastName?: string;
+
```
[the remark this answers is not in the document]
Comment (orr3) [reply src/tenant/tenant.graphql-model.ts:102]: I'd add a `?` to this

**Review by orr3 — APPROVED** [review]

**Pull request merged**